service/routes.py

```diff
@@ -46,6 +51,51 @@ def index():
     url = request.base_url + 'shopcarts'  # url_for('shopcarts')
     return jsonify(name='Shopcart Demo REST API Service', version='1.0', url=url, data=data), status.HTTP_200_OK
 
+######################################################################
+# ADD A NEW SHOPCART
+######################################################################
+@app.route("/shopcarts", methods=["POST"])
+def create_shopcarts():
+    """
+    Creates a Shopcart
+    This endpoint will create a Shopcart based the data in the body that is posted
+    """
+    app.logger.info("Request to create a shopcart")
+    check_content_type("application/json")
+    shopcart = Shopcart()
+    shopcart.deserialize(request.get_json())
```

Comment:
If the client sends a `create_time`, do we accept that?

Reply:
So I'll change the Create function to always use the current time